src/lightning_utilities/core/imports.py

```diff
@@ -4,10 +4,12 @@
 import importlib
 import operator
 from functools import lru_cache
+from importlib import metadata
```

Comment:
https://github.com/Lightning-AI/lightning/pull/15420 has failure because of this import. I don't think it's available in 3.7

Reply:
addressing it in #65